### DISCUSSION TITLE
Should we rename the Amstrad Mega PC to Amstrad PC7386SX?

### DISCUSSION THREAD

**Discussion by ghost** [86Box/86Box#2310] [General]
Title. Since we don't have plans to emulate the Mega Drive component of the computer, this leaves us with the PC component, which is actually a PC7386SX. Should we do this change?

**Comment by ArnoldUK** [comment]
I suppose it would be very easy to simply add a toolbar button that when clicked just shell executes your choice of the sega megadrive emulator :) there are many emulators already built for the sega megdrive. We should rename it the Amstrad Megabucks PC.

**Reply by ghost** [reply]
Except that the Mega Drive portion is out of scope for 86Box.

**Reply by ArnoldUK** [reply]
If you mean adding or building a full blown megadrive emulator, probably yes, but would be a nice feature. I know its jsut an addon card but 86Box is a PC emulator and I'd much prefer the developers worked on the PC side of things. Does MAME not have a Mega PC emulator ?

**Reply by OBattler** [reply]
I had the MegaPC (it was my first PC), and you couldn't use the Sega and PC parts at the same time. So 86Box only emulating the PC part is accurate in that when using the PC part, the Sega part for all intents and purposes didn't exist.

**Reply by ArnoldUK** [reply]
> MAME is still an arcade emulator

MAME also emualtes 8086 to 486 systems also and many 8-bit computers. But yes, I agree, would class it as an arcade emulation system.

**Reply by ArnoldUK** [reply]
> I had the MegaPC (it was my first PC), and you couldn't use the Sega and PC parts at the same time. So 86Box only emulating the PC part is accurate in that when using the PC part, the Sega part for all intents and purposes didn't exist.

Mee too, I had one and I remember trying to sell it back in the early 90's for about £50 but no bugger would buy it! I think the Amstrad badge put people off and most of me friends had mega drives anyway. I want it back now where ever it is :) Amstrad were always producing wacky designs with computers, I remember a PC that was build like a hifi system, I'd be surprised if the Amstrad R&D didnt think about adding a microwave to a PC.

**Reply by ghost** [reply]
> I had the MegaPC (it was my first PC), and you couldn't use the Sega and PC parts at the same time. So 86Box only emulating the PC part is accurate in that when using the PC part, the Sega part for all intents and purposes didn't exist.

So what do you say about this issue?

**Reply by OBattler** [reply]
I say that it's not needed because, as I explained, when using the PC part, the MegaPC was, for all intents and purposes, just that.

**Reply by ghost** [reply]
> I say that it's not needed because, as I explained, when using the PC part, the MegaPC was, for all intents and purposes, just that.

So, it was a PC7386SX. That's why I opened this discussion in the first place.

**Reply by ArnoldUK** [reply]
Yes, it was a PC7386SX, the mega drive part was an ISA card plugged into an ISA slot or propriety slot (AFAIK). The front slide cover suspended the PC cpu and switched to the mega drive card. Correct me if I was wrong, but were you are asking for the mega drive ROM to added along with the option of adding an emulated mega drive card to a PC configuration ? I see that to be a huge task to complete and is similar to asking for a WinTV card to be added so that a TV reciever can be emulated. Much easier to just pause 86Box and launch MAME or another sega emulator.

What would we call the Amstrad Mega Plus ? PC7486SLC ? What about the even rarer Sega Teradrive. Anyway, off topic, but I'd a much preferred if Amstrad built an Hybrid CPC 664 / PC than the mega drive. It would have encouraged the CPC users to upgrade to a PC and I bet sales would have doubled too, more so in the UK anyway. Here in the UK, Amstrad was a budget brand and was something you bought if a Sony, Toshiba or Hitachi was unaffordable.

**Reply by ghost** [reply]
> Correct me if I was wrong, but were you are asking for the mega drive ROM to added along with the option of adding an emulated mega drive card to a PC configuration ? I see that to be a huge task to complete and is similar to asking for a WinTV card to be added so that a TV reciever can be emulated. Much easier to just pause 86Box and launch MAME or another sega emulator.

Nope, the opposite. Since we don't emulate the Mega Drive, calling this the Mega PC is kind of a misnomer. That's why I suggest we rename this to PC7386SX, because what we emulate is just that.

**Reply by ArnoldUK** [reply]
The Mega PC is a PC7386SX under the hood, I dont see what your point is really?

**Reply by OBattler** [reply]
I really don't understand how "MegaPC" is a misnomer... yes, we only emulate the PC part, but that doesn't make it any less MegaPC. And are we really sure the PC7386SX is 100% identical, BIOS-wise? They could have taken the chance to update the BIOS with the MegaPC, for example.

**Reply by ghost** [reply]
> The Mega PC is a PC7386SX under the hood, I dont see what your point is really?

What??? That's what I've been saying the whole time...

**Reply by ArnoldUK** [reply]
But the name Mega PC is more recoginized than PC7386SX and like OBattler said, I'm sure the BIOS was different to cater for the Mega Drive card. Also, when the Mega PC was in "PC Mode" it was in a way very like a PC7386SX. Correct me if I'm wrong, but the only way to use the Mega Drive was by the front panel switch, AFAIK you could not access the Mega drive hardware from DOS or Windows. So how would you emulate the panel switch apart from adding a button to boot an external Mega Drive emulator and then blanking the screen of the emulated PC. Which is what I mentioned above. Then you have a fully emulated Mega PC.

**Reply by OBattler** [reply]
Not even necessarily to cater to the Mega Drive card (the BIOS of the PC part need not care about that), could have been simply Amstrad taking the opportunity of updating the BIOS to a slightly newer revision, something they then did with the 486SLC version as well. Now, I can't find a BIOS dump of the actual PC7386SX to compare this with.
